### ISSUE TITLE
Compilation error on import

### ISSUE THREAD

**Issue by jakob-lilliemarck** [datalogix/google-fonts-helper#34]
Im getting an odd error when trying to import as described in the README. Webpack 4 and Node 16 - any clues?

Does this package expect a filesystem and a running server? Is it possible to integrate with a SPA?

```
[Webpack] ERROR in ./node_modules/google-fonts-helper/dist/index.mjs 540:23-29
[Webpack] Can't import the named export '$fetch' from non EcmaScript module (only default export is available)
[Webpack]     at HarmonyImportSpecifierDependency._getErrors (/home/jakob/Projects/KlingitApp/node_modules/webpack/lib/dependencies/HarmonyImportSpecifierDependency.js:88:6)
[Webpack]     at HarmonyImportSpecifierDependency.getErrors (/home/jakob/Projects/KlingitApp/node_modules/webpack/lib/dependencies/HarmonyImportSpecifierDependency.js:68:16)
[Webpack]     at Compilation.reportDependencyErrorsAndWarnings (/home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compilation.js:1463:22)
[Webpack]     at /home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compilation.js:1258:10
[Webpack]     at AsyncSeriesHook.eval [as callAsync] (eval at create (/home/jakob/Projects/KlingitApp/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:15:1)
[Webpack]     at AsyncSeriesHook.lazyCompileHook (/home/jakob/Projects/KlingitApp/node_modules/tapable/lib/Hook.js:154:20)
[Webpack]     at Compilation.finish (/home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compilation.js:1253:28)
[Webpack]     at /home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compiler.js:672:17
[Webpack]     at _done (eval at create (/home/jakob/Projects/KlingitApp/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:9:1)
[Webpack]     at eval (eval at create (/home/jakob/Projects/KlingitApp/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:20:22)
[Webpack]     at /home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compilation.js:1185:12
[Webpack]     at /home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compilation.js:1097:9
[Webpack]     at processTicksAndRejections (node:internal/process/task_queues:78:11)
[Webpack] 
[Webpack] ERROR in ./node_modules/google-fonts-helper/dist/index.mjs 607:25-31
[Webpack] Can't import the named export '$fetch' from non EcmaScript module (only default export is available)
[Webpack]     at HarmonyImportSpecifierDependency._getErrors (/home/jakob/Projects/KlingitApp/node_modules/webpack/lib/dependencies/HarmonyImportSpecifierDependency.js:88:6)
[Webpack]     at HarmonyImportSpecifierDependency.getErrors (/home/jakob/Projects/KlingitApp/node_modules/webpack/lib/dependencies/HarmonyImportSpecifierDependency.js:68:16)
[Webpack]     at Compilation.reportDependencyErrorsAndWarnings (/home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compilation.js:1463:22)
[Webpack]     at /home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compilation.js:1258:10
[Webpack]     at AsyncSeriesHook.eval [as callAsync] (eval at create (/home/jakob/Projects/KlingitApp/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:15:1)
[Webpack]     at AsyncSeriesHook.lazyCompileHook (/home/jakob/Projects/KlingitApp/node_modules/tapable/lib/Hook.js:154:20)
[Webpack]     at Compilation.finish (/home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compilation.js:1253:28)
[Webpack]     at /home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compiler.js:672:17
[Webpack]     at _done (eval at create (/home/jakob/Projects/KlingitApp/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:9:1)
[Webpack]     at eval (eval at create (/home/jakob/Projects/KlingitApp/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:20:22)
[Webpack]     at /home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compilation.js:1185:12
[Webpack]     at /home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compilation.js:1097:9
[Webpack]     at processTicksAndRejections (node:internal/process/task_queues:78:11)
[Webpack] 
[Webpack] ERROR in ./node_modules/ohmyfetch/dist/node.mjs 38:36-45
[Webpack] Can't import the named export 'Headers' from non EcmaScript module (only default export is available)
[Webpack]     at HarmonyImportSpecifierDependency._getErrors (/home/jakob/Projects/KlingitApp/node_modules/webpack/lib/dependencies/HarmonyImportSpecifierDependency.js:88:6)
[Webpack]     at HarmonyImportSpecifierDependency.getErrors (/home/jakob/Projects/KlingitApp/node_modules/webpack/lib/dependencies/HarmonyImportSpecifierDependency.js:68:16)
[Webpack]     at Compilation.reportDependencyErrorsAndWarnings (/home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compilation.js:1463:22)
[Webpack]     at /home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compilation.js:1258:10
[Webpack]     at AsyncSeriesHook.eval [as callAsync] (eval at create (/home/jakob/Projects/KlingitApp/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:15:1)
[Webpack]     at AsyncSeriesHook.lazyCompileHook (/home/jakob/Projects/KlingitApp/node_modules/tapable/lib/Hook.js:154:20)
[Webpack]     at Compilation.finish (/home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compilation.js:1253:28)
[Webpack]     at /home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compiler.js:672:17
[Webpack]     at _done (eval at create (/home/jakob/Projects/KlingitApp/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:9:1)
[Webpack]     at eval (eval at create (/home/jakob/Projects/KlingitApp/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:20:22)
[Webpack]     at /home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compilation.js:1185:12
[Webpack]     at /home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compilation.js:1097:9
[Webpack]     at processTicksAndRejections (node:internal/process/task_queues:78:11)
[Webpack] 
[Webpack] ERROR in ./node_modules/google-fonts-helper/dist/index.mjs 706:2-10
[Webpack] Can't import the named export 'Hookable' from non EcmaScript module (only default export is available)
[Webpack]     at HarmonyImportSpecifierDependency._getErrors (/home/jakob/Projects/KlingitApp/node_modules/webpack/lib/dependencies/HarmonyImportSpecifierDependency.js:88:6)
[Webpack]     at HarmonyImportSpecifierDependency.getErrors (/home/jakob/Projects/KlingitApp/node_modules/webpack/lib/dependencies/HarmonyImportSpecifierDependency.js:68:16)
[Webpack]     at Compilation.reportDependencyErrorsAndWarnings (/home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compilation.js:1463:22)
[Webpack]     at /home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compilation.js:1258:10
[Webpack]     at AsyncSeriesHook.eval [as callAsync] (eval at create (/home/jakob/Projects/KlingitApp/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:15:1)
[Webpack]     at AsyncSeriesHook.lazyCompileHook (/home/jakob/Projects/KlingitApp/node_modules/tapable/lib/Hook.js:154:20)
[Webpack]     at Compilation.finish (/home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compilation.js:1253:28)
[Webpack]     at /home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compiler.js:672:17
[Webpack]     at _done (eval at create (/home/jakob/Projects/KlingitApp/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:9:1)
[Webpack]     at eval (eval at create (/home/jakob/Projects/KlingitApp/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:20:22)
[Webpack]     at /home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compilation.js:1185:12
[Webpack]     at /home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compilation.js:1097:9
[Webpack]     at processTicksAndRejections (node:internal/process/task_queues:78:11)
[Webpack] 
[Webpack] ERROR in ./node_modules/google-fonts-helper/dist/index.mjs 330:21-29
[Webpack] Can't import the named export 'basename' from non EcmaScript module (only default export is available)
[Webpack]     at HarmonyImportSpecifierDependency._getErrors (/home/jakob/Projects/KlingitApp/node_modules/webpack/lib/dependencies/HarmonyImportSpecifierDependency.js:88:6)
[Webpack]     at HarmonyImportSpecifierDependency.getErrors (/home/jakob/Projects/KlingitApp/node_modules/webpack/lib/dependencies/HarmonyImportSpecifierDependency.js:68:16)
[Webpack]     at Compilation.reportDependencyErrorsAndWarnings (/home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compilation.js:1463:22)
[Webpack]     at /home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compilation.js:1258:10
[Webpack]     at AsyncSeriesHook.eval [as callAsync] (eval at create (/home/jakob/Projects/KlingitApp/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:15:1)
[Webpack]     at AsyncSeriesHook.lazyCompileHook (/home/jakob/Projects/KlingitApp/node_modules/tapable/lib/Hook.js:154:20)
[Webpack]     at Compilation.finish (/home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compilation.js:1253:28)
[Webpack]     at /home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compiler.js:672:17
[Webpack]     at _done (eval at create (/home/jakob/Projects/KlingitApp/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:9:1)
[Webpack]     at eval (eval at create (/home/jakob/Projects/KlingitApp/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:20:22)
[Webpack]     at /home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compilation.js:1185:12
[Webpack]     at /home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compilation.js:1097:9
[Webpack]     at processTicksAndRejections (node:internal/process/task_queues:78:11)
[Webpack] 
[Webpack] ERROR in ./node_modules/google-fonts-helper/dist/index.mjs 442:19-28
[Webpack] Can't import the named export 'createURL' from non EcmaScript module (only default export is available)
[Webpack]     at HarmonyImportSpecifierDependency._getErrors (/home/jakob/Projects/KlingitApp/node_modules/webpack/lib/dependencies/HarmonyImportSpecifierDependency.js:88:6)
[Webpack]     at HarmonyImportSpecifierDependency.getErrors (/home/jakob/Projects/KlingitApp/node_modules/webpack/lib/dependencies/HarmonyImportSpecifierDependency.js:68:16)
[Webpack]     at Compilation.reportDependencyErrorsAndWarnings (/home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compilation.js:1463:22)
[Webpack]     at /home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compilation.js:1258:10
[Webpack]     at AsyncSeriesHook.eval [as callAsync] (eval at create (/home/jakob/Projects/KlingitApp/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:15:1)
[Webpack]     at AsyncSeriesHook.lazyCompileHook (/home/jakob/Projects/KlingitApp/node_modules/tapable/lib/Hook.js:154:20)
[Webpack]     at Compilation.finish (/home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compilation.js:1253:28)
[Webpack]     at /home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compiler.js:672:17
[Webpack]     at _done (eval at create (/home/jakob/Projects/KlingitApp/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:9:1)
[Webpack]     at eval (eval at create (/home/jakob/Projects/KlingitApp/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:20:22)
[Webpack]     at /home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compilation.js:1185:12
[Webpack]     at /home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compilation.js:1097:9
[Webpack]     at processTicksAndRejections (node:internal/process/task_queues:78:11)
[Webpack] 
[Webpack] ERROR in ./node_modules/google-fonts-helper/dist/index.mjs 324:16-23
[Webpack] Can't import the named export 'extname' from non EcmaScript module (only default export is available)
[Webpack]     at HarmonyImportSpecifierDependency._getErrors (/home/jakob/Projects/KlingitApp/node_modules/webpack/lib/dependencies/HarmonyImportSpecifierDependency.js:88:6)
[Webpack]     at HarmonyImportSpecifierDependency.getErrors (/home/jakob/Projects/KlingitApp/node_modules/webpack/lib/dependencies/HarmonyImportSpecifierDependency.js:68:16)
[Webpack]     at Compilation.reportDependencyErrorsAndWarnings (/home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compilation.js:1463:22)
[Webpack]     at /home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compilation.js:1258:10
[Webpack]     at AsyncSeriesHook.eval [as callAsync] (eval at create (/home/jakob/Projects/KlingitApp/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:15:1)
[Webpack]     at AsyncSeriesHook.lazyCompileHook (/home/jakob/Projects/KlingitApp/node_modules/tapable/lib/Hook.js:154:20)
[Webpack]     at Compilation.finish (/home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compilation.js:1253:28)
[Webpack]     at /home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compiler.js:672:17
[Webpack]     at _done (eval at create (/home/jakob/Projects/KlingitApp/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:9:1)
[Webpack]     at eval (eval at create (/home/jakob/Projects/KlingitApp/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:20:22)
[Webpack]     at /home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compilation.js:1185:12
[Webpack]     at /home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compilation.js:1097:9
[Webpack]     at processTicksAndRejections (node:internal/process/task_queues:78:11)
[Webpack] 
[Webpack] ERROR in ./node_modules/google-fonts-helper/dist/index.mjs 344:35-40
[Webpack] Can't import the named export 'posix' from non EcmaScript module (only default export is available)
[Webpack]     at HarmonyImportSpecifierDependency._getErrors (/home/jakob/Projects/KlingitApp/node_modules/webpack/lib/dependencies/HarmonyImportSpecifierDependency.js:88:6)
[Webpack]     at HarmonyImportSpecifierDependency.getErrors (/home/jakob/Projects/KlingitApp/node_modules/webpack/lib/dependencies/HarmonyImportSpecifierDependency.js:68:16)
[Webpack]     at Compilation.reportDependencyErrorsAndWarnings (/home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compilation.js:1463:22)
[Webpack]     at /home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compilation.js:1258:10
[Webpack]     at AsyncSeriesHook.eval [as callAsync] (eval at create (/home/jakob/Projects/KlingitApp/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:15:1)
[Webpack]     at AsyncSeriesHook.lazyCompileHook (/home/jakob/Projects/KlingitApp/node_modules/tapable/lib/Hook.js:154:20)
[Webpack]     at Compilation.finish (/home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compilation.js:1253:28)
[Webpack]     at /home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compiler.js:672:17
[Webpack]     at _done (eval at create (/home/jakob/Projects/KlingitApp/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:9:1)
[Webpack]     at eval (eval at create (/home/jakob/Projects/KlingitApp/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:20:22)
[Webpack]     at /home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compilation.js:1185:12
[Webpack]     at /home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compilation.js:1097:9
[Webpack]     at processTicksAndRejections (node:internal/process/task_queues:78:11)
[Webpack] 
[Webpack] ERROR in ./node_modules/google-fonts-helper/dist/index.mjs 525:26-33
[Webpack] Can't import the named export 'resolve' from non EcmaScript module (only default export is available)
[Webpack]     at HarmonyImportSpecifierDependency._getErrors (/home/jakob/Projects/KlingitApp/node_modules/webpack/lib/dependencies/HarmonyImportSpecifierDependency.js:88:6)
[Webpack]     at HarmonyImportSpecifierDependency.getErrors (/home/jakob/Projects/KlingitApp/node_modules/webpack/lib/dependencies/HarmonyImportSpecifierDependency.js:68:16)
[Webpack]     at Compilation.reportDependencyErrorsAndWarnings (/home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compilation.js:1463:22)
[Webpack]     at /home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compilation.js:1258:10
[Webpack]     at AsyncSeriesHook.eval [as callAsync] (eval at create (/home/jakob/Projects/KlingitApp/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:15:1)
[Webpack]     at AsyncSeriesHook.lazyCompileHook (/home/jakob/Projects/KlingitApp/node_modules/tapable/lib/Hook.js:154:20)
[Webpack]     at Compilation.finish (/home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compilation.js:1253:28)
[Webpack]     at /home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compiler.js:672:17
[Webpack]     at _done (eval at create (/home/jakob/Projects/KlingitApp/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:9:1)
[Webpack]     at eval (eval at create (/home/jakob/Projects/KlingitApp/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:20:22)
[Webpack]     at /home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compilation.js:1185:12
[Webpack]     at /home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compilation.js:1097:9
[Webpack]     at processTicksAndRejections (node:internal/process/task_queues:78:11)
[Webpack] 
[Webpack] ERROR in ./node_modules/google-fonts-helper/dist/index.mjs 636:29-36
[Webpack] Can't import the named export 'resolve' from non EcmaScript module (only default export is available)
[Webpack]     at HarmonyImportSpecifierDependency._getErrors (/home/jakob/Projects/KlingitApp/node_modules/webpack/lib/dependencies/HarmonyImportSpecifierDependency.js:88:6)
[Webpack]     at HarmonyImportSpecifierDependency.getErrors (/home/jakob/Projects/KlingitApp/node_modules/webpack/lib/dependencies/HarmonyImportSpecifierDependency.js:68:16)
[Webpack]     at Compilation.reportDependencyErrorsAndWarnings (/home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compilation.js:1463:22)
[Webpack]     at /home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compilation.js:1258:10
[Webpack]     at AsyncSeriesHook.eval [as callAsync] (eval at create (/home/jakob/Projects/KlingitApp/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:15:1)
[Webpack]     at AsyncSeriesHook.lazyCompileHook (/home/jakob/Projects/KlingitApp/node_modules/tapable/lib/Hook.js:154:20)
[Webpack]     at Compilation.finish (/home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compilation.js:1253:28)
[Webpack]     at /home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compiler.js:672:17
[Webpack]     at _done (eval at create (/home/jakob/Projects/KlingitApp/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:9:1)
[Webpack]     at eval (eval at create (/home/jakob/Projects/KlingitApp/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:20:22)
[Webpack]     at /home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compilation.js:1185:12
[Webpack]     at /home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compilation.js:1097:9
[Webpack]     at processTicksAndRejections (node:internal/process/task_queues:78:11)
[Webpack] 
[Webpack] ERROR in ./node_modules/google-fonts-helper/dist/index.mjs 420:29-39
[Webpack] Can't import the named export 'resolveURL' from non EcmaScript module (only default export is available)
[Webpack]     at HarmonyImportSpecifierDependency._getErrors (/home/jakob/Projects/KlingitApp/node_modules/webpack/lib/dependencies/HarmonyImportSpecifierDependency.js:88:6)
[Webpack]     at HarmonyImportSpecifierDependency.getErrors (/home/jakob/Projects/KlingitApp/node_modules/webpack/lib/dependencies/HarmonyImportSpecifierDependency.js:68:16)
[Webpack]     at Compilation.reportDependencyErrorsAndWarnings (/home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compilation.js:1463:22)
[Webpack]     at /home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compilation.js:1258:10
[Webpack]     at AsyncSeriesHook.eval [as callAsync] (eval at create (/home/jakob/Projects/KlingitApp/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:15:1)
[Webpack]     at AsyncSeriesHook.lazyCompileHook (/home/jakob/Projects/KlingitApp/node_modules/tapable/lib/Hook.js:154:20)
[Webpack]     at Compilation.finish (/home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compilation.js:1253:28)
[Webpack]     at /home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compiler.js:672:17
[Webpack]     at _done (eval at create (/home/jakob/Projects/KlingitApp/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:9:1)
[Webpack]     at eval (eval at create (/home/jakob/Projects/KlingitApp/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:20:22)
[Webpack]     at /home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compilation.js:1185:12
[Webpack]     at /home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compilation.js:1097:9
[Webpack]     at processTicksAndRejections (node:internal/process/task_queues:78:11)
[Webpack] 
[Webpack] ERROR in ./node_modules/ohmyfetch/dist/chunks/fetch.mjs 199:32-40
[Webpack] Can't import the named export 'withBase' from non EcmaScript module (only default export is available)
[Webpack]     at HarmonyImportSpecifierDependency._getErrors (/home/jakob/Projects/KlingitApp/node_modules/webpack/lib/dependencies/HarmonyImportSpecifierDependency.js:88:6)
[Webpack]     at HarmonyImportSpecifierDependency.getErrors (/home/jakob/Projects/KlingitApp/node_modules/webpack/lib/dependencies/HarmonyImportSpecifierDependency.js:68:16)
[Webpack]     at Compilation.reportDependencyErrorsAndWarnings (/home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compilation.js:1463:22)
[Webpack]     at /home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compilation.js:1258:10
[Webpack]     at AsyncSeriesHook.eval [as callAsync] (eval at create (/home/jakob/Projects/KlingitApp/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:15:1)
[Webpack]     at AsyncSeriesHook.lazyCompileHook (/home/jakob/Projects/KlingitApp/node_modules/tapable/lib/Hook.js:154:20)
[Webpack]     at Compilation.finish (/home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compilation.js:1253:28)
[Webpack]     at /home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compiler.js:672:17
[Webpack]     at _done (eval at create (/home/jakob/Projects/KlingitApp/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:9:1)
[Webpack]     at eval (eval at create (/home/jakob/Projects/KlingitApp/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:20:22)
[Webpack]     at /home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compilation.js:1185:12
[Webpack]     at /home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compilation.js:1097:9
[Webpack]     at processTicksAndRejections (node:internal/process/task_queues:78:11)
[Webpack] 
[Webpack] ERROR in ./node_modules/google-fonts-helper/dist/index.mjs 420:9-18
[Webpack] Can't import the named export 'withHttps' from non EcmaScript module (only default export is available)
[Webpack]     at HarmonyImportSpecifierDependency._getErrors (/home/jakob/Projects/KlingitApp/node_modules/webpack/lib/dependencies/HarmonyImportSpecifierDependency.js:88:6)
[Webpack]     at HarmonyImportSpecifierDependency.getErrors (/home/jakob/Projects/KlingitApp/node_modules/webpack/lib/dependencies/HarmonyImportSpecifierDependency.js:68:16)
[Webpack]     at Compilation.reportDependencyErrorsAndWarnings (/home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compilation.js:1463:22)
[Webpack]     at /home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compilation.js:1258:10
[Webpack]     at AsyncSeriesHook.eval [as callAsync] (eval at create (/home/jakob/Projects/KlingitApp/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:15:1)
[Webpack]     at AsyncSeriesHook.lazyCompileHook (/home/jakob/Projects/KlingitApp/node_modules/tapable/lib/Hook.js:154:20)
[Webpack]     at Compilation.finish (/home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compilation.js:1253:28)
[Webpack]     at /home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compiler.js:672:17
[Webpack]     at _done (eval at create (/home/jakob/Projects/KlingitApp/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:9:1)
[Webpack]     at eval (eval at create (/home/jakob/Projects/KlingitApp/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:20:22)
[Webpack]     at /home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compilation.js:1185:12
[Webpack]     at /home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compilation.js:1097:9
[Webpack]     at processTicksAndRejections (node:internal/process/task_queues:78:11)
[Webpack] 
[Webpack] ERROR in ./node_modules/google-fonts-helper/dist/index.mjs 420:19-28
[Webpack] Can't import the named export 'withQuery' from non EcmaScript module (only default export is available)
[Webpack]     at HarmonyImportSpecifierDependency._getErrors (/home/jakob/Projects/KlingitApp/node_modules/webpack/lib/dependencies/HarmonyImportSpecifierDependency.js:88:6)
[Webpack]     at HarmonyImportSpecifierDependency.getErrors (/home/jakob/Projects/KlingitApp/node_modules/webpack/lib/dependencies/HarmonyImportSpecifierDependency.js:68:16)
[Webpack]     at Compilation.reportDependencyErrorsAndWarnings (/home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compilation.js:1463:22)
[Webpack]     at /home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compilation.js:1258:10
[Webpack]     at AsyncSeriesHook.eval [as callAsync] (eval at create (/home/jakob/Projects/KlingitApp/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:15:1)
[Webpack]     at AsyncSeriesHook.lazyCompileHook (/home/jakob/Projects/KlingitApp/node_modules/tapable/lib/Hook.js:154:20)
[Webpack]     at Compilation.finish (/home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compilation.js:1253:28)
[Webpack]     at /home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compiler.js:672:17
    at _done (eval at create (/home/jakob/Projects/KlingitApp/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:9:1)
[Webpack]     at eval (eval at create (/home/jakob/Projects/KlingitApp/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:20:22)
[Webpack]     at /home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compilation.js:1185:12
[Webpack]     at /home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compilation.js:1097:9
[Webpack]     at processTicksAndRejections (node:internal/process/task_queues:78:11)
[Webpack] 
[Webpack] ERROR in ./node_modules/ohmyfetch/dist/chunks/fetch.mjs 203:32-41
[Webpack] Can't import the named export 'withQuery' from non EcmaScript module (only default export is available)
[Webpack]     at HarmonyImportSpecifierDependency._getErrors (/home/jakob/Projects/KlingitApp/node_modules/webpack/lib/dependencies/HarmonyImportSpecifierDependency.js:88:6)
[Webpack]     at HarmonyImportSpecifierDependency.getErrors (/home/jakob/Projects/KlingitApp/node_modules/webpack/lib/dependencies/HarmonyImportSpecifierDependency.js:68:16)
[Webpack]     at Compilation.reportDependencyErrorsAndWarnings (/home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compilation.js:1463:22)
[Webpack]     at /home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compilation.js:1258:10
[Webpack]     at AsyncSeriesHook.eval [as callAsync] (eval at create (/home/jakob/Projects/KlingitApp/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:15:1)
[Webpack]     at AsyncSeriesHook.lazyCompileHook (/home/jakob/Projects/KlingitApp/node_modules/tapable/lib/Hook.js:154:20)
[Webpack]     at Compilation.finish (/home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compilation.js:1253:28)
[Webpack]     at /home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compiler.js:672:17
[Webpack]     at _done (eval at create (/home/jakob/Projects/KlingitApp/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:9:1)
[Webpack]     at eval (eval at create (/home/jakob/Projects/KlingitApp/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:20:22)
[Webpack]     at /home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compilation.js:1185:12
[Webpack]     at /home/jakob/Projects/KlingitApp/node_modules/webpack/lib/Compilation.js:1097:9
[Webpack]     at processTicksAndRejections (node:internal/process/task_queues:78:11)
[Webpack] 
[Webpack] ERROR in ./node_modules/node-fetch-native/dist/index.cjs 69:2
[Webpack] Module parse failed: Unexpected character '#' (69:2)
[Webpack] You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
[Webpack] |  */
[Webpack] | class BlobDataItem {
[Webpack] >   #path
[Webpack] |   #start
[Webpack] | 

```

**Comment by ricardogobbosouza** [comment]
Hi @jakob-lilliemarck 

Could you provide a repository to reproduce?

**Comment by frederik** [comment]
@ricardogobbosouza  Just thought I'd leave this here. Pinning del to 6.1.1 instead of 7.0 solves the issues temporarily

from my test project's package.json
```
{
  [..]
  "dependencies": {
    "google-fonts-helper": "^3.2.2"
  },
  "resolutions": {
    "del": "6.1.1"
  }
}

```

Happens with any file if I just copy the import statement and then use download.

**Comment by ricardogobbosouza** [comment]
Fixed on [v3.2.3](https://github.com/datalogix/google-fonts-helper/tree/v3.2.3)
